fs-ansible/roles/lustre/tasks/wipe.yaml

```diff
@@ -17,4 +17,20 @@
     device: "/dev/{{ item }}"
     number: 2
     state: absent
-  loop: "{{ (mdts.keys() + osts.keys()) | unique }}"
\ No newline at end of file
+  loop: "{{ (mdts.keys() + osts.keys()) | unique }}"
+
+- name: Wait for MDT partition to disappear from /dev
+  wait_for:
+    path: "/dev/{{ item }}p1"
+    state: absent
+    timeout: 120
+    sleep: 5
+  loop: "{{ mdts.keys() }}"
```

Comment:
I would rather we used the same loop as above, as this will miss out waiting for some of the partitions to go, which will still cause problems as we always try to create them.

Reply:
basically we need to loop over both mdts and osts to make sure we delete all the partitions created by this loop:
https://github.com/RSE-Cambridge/data-acc/blob/60dd13248a2fffbd0749f6af9e646ce210ed98cd/fs-ansible/roles/lustre/tasks/format.yaml#L12

Reply:
I see, done.